packages/strapi-plugin-upload/admin/src/components/SortList/index.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import getTrad from '../../utils/getTrad';
+
+import Wrapper from './Wrapper';
+import SortListItem from './SortListItem';
+
+const SortList = ({ isShown, list, onClick, selected }) => {
+  return (
+    <Wrapper isShown={isShown}>
+      {Object.keys(list).map(item => {
+        return (
+          <SortListItem
```

Comment:
Your previous implementation of the List was correct! You need to extract the `SortListItem` :

```
return <SortListItem {...item} selectedItem={selectedItem} onClick={onClick} />
```